README.md

```diff
@@ -0,0 +1,29 @@
+# TNoodle Samples
+
+In this repository you will find numerous demonstrations which show you how to embed [TNoodle](https://github.com/thewca/tnoodle) into real-life projects.
+
+
+## List of sample projects
+
+TNoodle is itself written in Java, so to reuse the JAR artifacts all sample projects are Java-related.
+As such, every folder is structured like a standard [Gradle](https://gradle.com) build. The build files are written in the type-safe `Kotlin` dialect.
+
+Every sub-project has its individual artifact configuration and `build.gradle` file. Furthermore, there is a central `buildSrc` folder,
+which is automatically sourced by Gradle. It contains common code and shared configuration setups.
+
+
+### tnoodle-android
+
+A sample Android app with a drawer that lets you choose between several puzzles and generates scrambles for them.
+
+Note that in order for this setup to compile, you will have to **install the Android SDK** on your local machine.
+Steps and instructions for this can be found in the main [TNoodle README](https://github.com/thewca/tnoodle/blob/master/README.md) and at [https://developer.android.com/studio](https://developer.android.com/studio)
```

Comment:
Can we remove the android sdk instructions from the TNoodle readme and put them here instead?

Reply:
Yes, fixed. They will only be actually removed at the TNoodle main repo once this has been merged upstream (see also my answer to your Q1)